modules/templates/global-elements.xml

```diff
@@ -362,6 +362,230 @@ $(target).find('.date').each(function(_idx, dateEl){
+            </property>
+            <property id="shadowBlur" type="number">
+                <title>Shadow Blur</title>
+                <default>2</default>
+                <visibility>
+                    <test>
+                        <condition field="textShadow" type="eq">1</condition>
+                    </test>
+                </visibility>
+            </property>
+            <property id="horizontalAlign" type="dropdown">
+                <title>Horizontal Align</title>
+                <default>center</default>
+                <options>
+                    <option name="flex-start">Left</option>
+                    <option name="center">Center</option>
+                    <option name="flex-end">Right</option>
+                </options>
+            </property>
+            <property id="verticalAlign" type="dropdown">
+                <title>Vertical Align</title>
+                <default>center</default>
+                <options>
+                    <option name="flex-start">Top</option>
+                    <option name="center">Middle</option>
+                    <option name="flex-end">Bottom</option>
+                </options>
+            </property>
+        </properties>
+        <stencil>
+            <hbs><![CDATA[
+<div class="global-elements-date-advanced"
+    style="
+    display: flex;
+    {{#if fontFamily}}font-family: {{fontFamily}};{{/if}}
+    {{#if fontSize}}font-size: {{fontSize}}px;{{/if}}
+    {{#if lineHeight}}line-height: {{lineHeight}};{{/if}}
+    {{#if fontColor}}color: {{fontColor}};{{/if}}
+    {{#if bold}}font-weight: bold;{{/if}}
+    {{#if italics}}font-style: italic;{{/if}}
+    {{#if underline}}text-decoration: underline;{{/if}}
+    {{#if showOverflow}}overflow: visible;{{else}}overflow: hidden;{{/if}}
+    {{#if horizontalAlign}}justify-content: {{horizontalAlign}};{{/if}}
+    {{#if verticalAlign}}align-items: {{verticalAlign}};{{/if}}
+    {{#if textWrap}}white-space: break-spaces;{{else}}white-space: nowrap;{{/if}}
+    {{#if textShadow}}text-shadow: {{shadowX}}px {{shadowY}}px {{shadowBlur}}px {{textShadowColor}};{{/if}}
+    width: 100%; height: 100%;"
+>
+<div class="date-advanced"></div>
+</div>
+            ]]></hbs>
+        </stencil>
+        <onTemplateRender><![CDATA[
+var $dateEl = $(target).find('.date-advanced');
+var useCurrentDate = (properties.currentDate == 1);
+
+if (useCurrentDate) {
+    // Use current date
+
+    var offset = properties.offset || 0;
+    // Clear previous interval if exists
+    if (window['updateInterval_' + id]) {
+        clearInterval(window['updateInterval_' + id]);
+    }
+
+    // Update every second
+    window['updateInterval_' + id] = setInterval(function() {
+        // If element was removed, clear interval
+        if(!$dateEl.is(':visible')) {
+            clearInterval(window['updateInterval_' + id]);
+            return;
+        }
+
+        // Update date value
+        var currentDate = moment().add(offset, "m");
```

Comment:
`offset` isn't defined?

Reply:
`offset` is defined on line 538, but since it's only used on the `setInterval`, I can always move it to that scope.

Reply:
No that's ok, I understand now! 👍